Masquerade4997Code/src/main/java/org/firstinspires/ftc/teamcode/Robots/MarkOne/Autonomous/Blue/BlueBuildStoneAuto.java

```diff
@@ -20,10 +20,8 @@ public void runLinearOpMode() throws InterruptedException{
         robot.init(hardwareMap);
         robot.initializeAutonomous();
 
-        //while(!opModeIsActive()) {
-            dash.create("Hello");
-            dash.update();
-        //}
+        dash.create("Hello");
+        dash.update();
 
```

Comment:
Nope. Use the while !opModeIsActive

Reply:
It doesn’t change though

Reply:
While !opModeIsActive is standard. Use it